x-pack/plugins/security_solution/public/entity_analytics/components/entity_details_flyout/tabs/risk_inputs/risk_inputs_tab.tsx

```diff
@@ -183,79 +181,171 @@ export const RiskInputsTab = ({ entityType, entityName }: RiskInputsTabProps) =>
         <h3>
           <FormattedMessage
-            id="xpack.securitySolution.flyout.entityDetails.riskInputs.assetCriticalityTitle"
-            defaultMessage="Asset Criticality"
+            id="xpack.securitySolution.flyout.entityDetails.riskInputs.contextsTitle"
+            defaultMessage="Contexts"
           />
         </h3>
       </EuiTitle>
       <EuiSpacer size="xs" />
-      <EuiText size="xs" color="subdued">
-        <FormattedMessage
-          id="xpack.securitySolution.flyout.entityDetails.riskInputs.assetCriticalityDescription"
-          defaultMessage="The criticality assigned at the time of the risk score calculation."
-        />
-      </EuiText>
-      <EuiSpacer size="s" />
-      <AssetCriticalityBadgeAllowMissing
-        criticalityLevel={criticalityLevel}
-        dataTestSubj="risk-inputs-asset-criticality-badge"
+      <EuiInMemoryTable
+        compressed={true}
+        loading={loading}
+        columns={contextColumns}
+        items={[
+          {
+            field: (
+              <FormattedMessage
+                id="xpack.securitySolution.flyout.entityDetails.riskInputs.assetCriticalityField"
+                defaultMessage="Asset Criticality Level"
+              />
+            ),
+            value: (
+              <AssetCriticalityBadgeAllowMissing
+                criticalityLevel={criticality.level}
+                dataTestSubj="risk-inputs-asset-criticality-badge"
+              />
+            ),
+            contribution: (criticality.contribution || 0).toFixed(2),
+          },
+        ]}
       />
-
-      <EuiSpacer size="m" />
     </>
   );
 };
 
-RiskInputsTab.displayName = 'RiskInputsTab';
+interface ContextRow {
+  field: ReactNode;
+  value: ReactNode;
+  contribution: string;
+}
+
+const contextColumns: Array<EuiBasicTableColumn<ContextRow>> = [
+  {
+    field: 'field',
+    name: (
+      <FormattedMessage
+        id="xpack.securitySolution.flyout.entityDetails.riskInputs.actionsColumn"
+        defaultMessage="Field"
+      />
+    ),
+    width: '30%',
+    render: (field: ContextRow['field']) => field,
+  },
+  {
+    field: 'value',
+    name: (
+      <FormattedMessage
+        id="xpack.securitySolution.flyout.entityDetails.riskInputs.actionsColumn"
+        defaultMessage="Value"
+      />
+    ),
+    width: '30%',
+    render: (val: ContextRow['value']) => val,
+  },
+  {
+    field: 'contribution',
+    width: '30%',
+    align: 'right',
+    name: (
+      <FormattedMessage
+        id="xpack.securitySolution.flyout.entityDetails.riskInputs.actionsColumn"
+        defaultMessage="Contribution"
+      />
+    ),
+    render: (score: ContextRow['contribution']) => score,
+  },
+];
+
+interface AlertInputMessageProps {
+  totals: {
+    count: number;
+    score: number;
+  };
+  inputs: {
+    count: number;
+    score: number;
+  };
+}
+const AlertInputMessage: React.FC<AlertInputMessageProps> = ({ totals, inputs }) => {
```

Comment:
I didnt guess what this component did from its name, ExtraAlertsMessage maybe?

Reply:
yeah, nice, I like that more too